alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2967,6 +2967,39 @@ public static function addDeliveryTagToEntry(entry $entry)
 			$entry->save();
 		}
 	}
+
+	public static function handleDelayedNotification($object)
+	{
+
+		$delayedLockedJobs = match (true)
+		{
+			$object instanceof entry => BatchJobLockPeer::retrieveByEntryIdAndStatus($object->getEntryId(), BatchJob::BATCHJOB_STATUS_DELAYED)
```

Comment:
this will throw Fatal error if the function is called with an object that is not of type `entry`
while right now this is not possible, because it is only called from within a block that only handles entries - it might in the future if someone will call it from another context.
worth either adding a fallback like:
```
true => []
```
write the code differently without using `match`